api/client/service.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+package nighthawk.client;
+
+import "api/client/options.proto";
+import "api/client/output.proto";
+
+message StartRequest {
+  CommandLineOptions options = 1;
+}
+
+/*
+TODO(oschaaf): Not implemented yet.
+*/
+message UpdateRequest {
+}
+
+/*
+TODO(oschaaf): Not implemented yet.
+*/
+message CancellationRequest {
+}
+
+message ExecutionRequest {
+  oneof command_specific_options {
+    StartRequest start_request = 1;
+    UpdateRequest update_request = 2;
+    CancellationRequest cancellation_request = 3;
+  }
+}
+
+/*
```

Comment:
Nit, can you use `//` comments even for blocks, similar to Envoy style.

Reply:
Fixed